src/olympia/constants/base.py

```diff
@@ -224,7 +224,8 @@
 MIN_TAG_LENGTH = 2
 MAX_CATEGORIES = 2
 VALID_CONTRIBUTION_DOMAINS = (
-    'paypal.me', 'paypal.com', 'patreon.com', 'micropayment.de'
+    'donate.mozilla.org', 'paypal.me', 'paypal.com', 'patreon.com',
+    'micropayment.de'
```

Comment:
is there some logic to the order?  (you put `donate.mozilla.org` at the front, but alphabetically `micropayment.de` should be next, etc)

Reply:
Kinda. I tried to order them by how important they felt to me :) Maybe it's best to do it alphabetically now...

Reply:
I suppose it's nice `donate.mozilla.org` is at the front.  The rest, meh :)